README.md

```diff
@@ -361,6 +361,82 @@ secrets:
       name: xwiki-db-root-password
 ```
 
+## Configuring clustering
+
+Read about [setting communication channels](https://www.xwiki.org/xwiki/bin/view/Documentation/AdminGuide/Clustering/).
+
+#### Docker Compose example
+
+```yaml
+
+version: '2'
+networks:
+  bridge:
+    driver: bridge
+services:
+  web:
+    build: .
+    container_name: xwiki-postgres-tomcat-web
+    depends_on:
+      - db
+    ports:
+      - "8080:8080"
+    environment:
+      - XWIKI_VERSION=${XWIKI_VERSION}
+      - DB_USER=${DB_USER}
+      - DB_PASSWORD=${DB_PASSWORD}
+      - DB_DATABASE=${DB_DATABASE}
+      - DB_HOST=xwiki-postgres-db
+      - CLUSTER=true
+      - CLUSTER_CHANNEL=udp
+    volumes:
+      - xwiki-data-cache:/usr/local/xwiki/data/cache
+      - xwiki-data-jobs:/usr/local/xwiki/data/jobs
+      - xwiki-data-extension:/usr/local/xwiki/data/extension
```

Comment:
I don't understand why you created 3 mounts and not just one for the perm dir. Could you explain?

Reply:
commented below